src/goto-instrument/dump_c.cpp

```diff
@@ -526,7 +526,7 @@ void dump_ct::convert_compound(
       if(t.get_width()<=config.ansi_c.long_long_int_width)
         struct_body << "long long int " << comp_name
           << " : " << t.get_width();
-      else if(language->id()=="cpp")
+      else if(mode == ID_cpp)
```

Comment:
Doesn't seem to correspond with the PR message.  I'm fine with them going in as a minor fix but just wanted to flag it in case it was a mistake.